packages/@sanity/form-builder/src/inputs/PortableText/text/textStyles.tsx

```diff
@@ -2,45 +2,50 @@ import {Heading, Text} from '@sanity/ui'
 import React from 'react'
 import styled from 'styled-components'
 
+// Without this container, editing with Android breaks due to how Text is styled via responsiveFont in @sanity/ui
+const TextContainer = styled('div')`
+  display: block;
+`
+
```

Comment:
Seems like this can either be replaced by 
```tsx
const TextContainer = styled.div``
```

or simple a `<div>`, since `div` is `display: block` by default.